themes/default/content/docs/intro/concepts/resources/options/protect.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Protect"
+meta_desc: An in depth look at Pulumi resources and their usage.
+menu:
+  intro:
+    parent: options
+    weight: 9
+---
+
+The `protect` resource option marks a resource as protected. A protected resource cannot be deleted directly. Instead, you must first set `protect: false` and run `pulumi up`. Then you can delete the resource by removing the line of code or by running `pulumi destroy`. The default is to inherit this value from the parent resource, and `false` for resources without a parent.
```

Comment:
Do you think it'd make sense to briefly mention/link to `pulumi state unprotect` here as well?